king_phisher/client/plugins.py

```diff
@@ -543,3 +552,208 @@ class ClientPluginManager(plugins.PluginManagerBase):
 	_plugin_klass = ClientPlugin
 	def __init__(self, path, application):
 		super(ClientPluginManager, self).__init__(path, (application,))
+
+class RepoCache(object):
```

Comment:
Also, could this entire class just be a `namedtuple` since it is infact a `cache` I would assume that it would never be updated unless the repositories information itself were updated in which case you could reinitialize a new instance.